graphrag_eval/steps/__init__.py

```diff
@@ -128,6 +130,19 @@ def get_steps_evaluation_result_dict(reference: dict, target: dict) -> dict:
             act_step.update(result)
     if "reference_steps" in reference:
         ref_steps = reference["reference_steps"]
-        steps_score = evaluate_steps(ref_steps, act_steps)
+        matches = get_steps_matches(ref_steps, act_steps)
+        steps_score = evaluate_steps(ref_steps, act_steps, matches)
         eval_result["steps_score"] = steps_score
+        for ref_group_idx, ref_match_idx, act_idx, _ in matches:
+            ref_step = ref_steps[ref_group_idx][ref_match_idx]
+            act_step = act_steps[act_idx]
+            if ref_step["name"] == "retrieval":
+                if "output" in ref_step:
```

Comment:
I'm again confused. Isn't the output mandatory in this case? Also, the added changes to the README lack output

Reply:
Thanks. Removed the `if`-statement

Reply:
Added `output` field in README example input and output